enterprise/cli/server_dbcrypt.go

```diff
@@ -0,0 +1,331 @@
+				return err
+			}
+
+			sqlDB, err := cli.ConnectToPostgres(inv.Context(), logger, "postgres", flags.PostgresURL)
+			if err != nil {
+				return xerrors.Errorf("connect to postgres: %w", err)
+			}
+			defer func() {
+				_ = sqlDB.Close()
+			}()
+			logger.Info(ctx, "connected to postgres")
+			if err := dbcrypt.Rotate(ctx, logger, sqlDB, ciphers); err != nil {
+				return xerrors.Errorf("rotate ciphers: %w", err)
+			}
+			logger.Info(ctx, "operation completed successfully")
+			return nil
+		},
+	}
+	flags.attach(&cmd.Options)
+	return cmd
+}
+
+func (*RootCmd) dbcryptDecryptCmd() *clibase.Cmd {
+	var flags decryptFlags
+	cmd := &clibase.Cmd{
+		Use:   "decrypt",
+		Short: "Decrypt a previously encrypted database.",
+		Handler: func(inv *clibase.Invocation) error {
+			ctx, cancel := context.WithCancel(inv.Context())
+			defer cancel()
+			logger := slog.Make(sloghuman.Sink(inv.Stdout))
+			if ok, _ := inv.ParsedFlags().GetBool("verbose"); ok {
+				logger = logger.Leveled(slog.LevelDebug)
+			}
+
+			if err := flags.valid(); err != nil {
+				return err
+			}
+
+			ks := make([][]byte, 0, len(flags.Keys))
+			for _, k := range flags.Keys {
+				dk, err := base64.StdEncoding.DecodeString(k)
+				if err != nil {
+					return xerrors.Errorf("decode key: %w", err)
+				}
+				ks = append(ks, dk)
+			}
+
+			ciphers, err := dbcrypt.NewCiphers(ks...)
+			if err != nil {
+				return xerrors.Errorf("create ciphers: %w", err)
+			}
+
+			if _, err := cliui.Prompt(inv, cliui.PromptOptions{
+				Text:      "This will decrypt all encrypted data in the database. Are you sure you want to continue?",
+				IsConfirm: true,
+			}); err != nil {
+				return err
+			}
+
+			sqlDB, err := cli.ConnectToPostgres(inv.Context(), logger, "postgres", flags.PostgresURL)
+			if err != nil {
+				return xerrors.Errorf("connect to postgres: %w", err)
+			}
+			defer func() {
+				_ = sqlDB.Close()
+			}()
+			logger.Info(ctx, "connected to postgres")
+			if err := dbcrypt.Decrypt(ctx, logger, sqlDB, ciphers); err != nil {
+				return xerrors.Errorf("rotate ciphers: %w", err)
+			}
+			logger.Info(ctx, "operation completed successfully")
+			return nil
+		},
+	}
+	flags.attach(&cmd.Options)
+	return cmd
+}
+
+func (*RootCmd) dbcryptDeleteCmd() *clibase.Cmd {
+	var flags deleteFlags
+	cmd := &clibase.Cmd{
+		Use:   "delete",
+		Short: "Delete all encrypted data from the database. THIS IS A DESTRUCTIVE OPERATION.",
+		Handler: func(inv *clibase.Invocation) error {
+			ctx, cancel := context.WithCancel(inv.Context())
+			defer cancel()
+			logger := slog.Make(sloghuman.Sink(inv.Stdout))
+			if ok, _ := inv.ParsedFlags().GetBool("verbose"); ok {
+				logger = logger.Leveled(slog.LevelDebug)
+			}
+
+			if err := flags.valid(); err != nil {
+				return err
+			}
+
+			if _, err := cliui.Prompt(inv, cliui.PromptOptions{
+				Text:      "This will delete all encrypted data from the database. Are you sure you want to continue?",
```

Comment:
Should say what data will actually be deleted here and what the consequences are.